java/ql/src/semmle/code/java/frameworks/SnakeYaml.qll

```diff
@@ -71,7 +71,7 @@ private class SnakeYamlParse extends MethodAccess {
   SnakeYamlParse() {
     exists(Method m |
       m.getDeclaringType() instanceof Yaml and
-      (m.hasName("load") or m.hasName("loadAll") or m.hasName("loadAs") or m.hasName("parse")) and
+      (m.hasName("compose") or m.hasName("composeAll") or m.hasName("load") or m.hasName("loadAll") or m.hasName("loadAs") or m.hasName("parse")) and
```

Comment:
We can simplify a bit:
```suggestion
      m.hasName(["compose", "composeAll", "load", "loadAll", "loadAs", "parse"]) and
```

Reply:
Cool, knew about `m.getName() in [...]` but not that you could pass an array to `hasName`

Reply:
It's not an array - it's just multiple values. It's a general language feature that works everywhere. It means the same as introducing a new variable and giving that variable multiple possible values using `or`.